doc/components.md

```diff
@@ -25,6 +25,43 @@ Get all entries of the view-models 'userRepository' repository property and popu
 ></association-select>
 ```
 
+## Basic example, with placeholder settings
+
+The same as the basic example above, with:
+
+* hiding the placeholder option (`value=0`):
```

Comment:
though thinking of it. maybe just mention once or twice at the right place that placeholder means value 0 and then just use placeholder. that'll make it more readable